doc_user/UserManual.adoc

```diff
@@ -630,7 +631,32 @@ to the expiring date. If the first selected lot does not contain a high enough q
 movement, several discharging movements may be generated. Ask the Administrator or check the _Administrator’s Guide_
 for more information.
 
-<<<
+[#delete-last-stock-movement]
+===== 4.2.2.3 Delete Last Stock Movement (*[.underline]##D##elete*)
+To delete the last movement, select first the row that you want to delete and press the *[.underline]##D##elete* button in the *_Stock Movement Browser_* window. If it is not the last movement you will see a *_Error_* window:
+
+image:image250.png[Error]
+
+if you don't choosed any row and:
+
+image:image252.png[Error]
+
+if you choosed more than one row.
+
+But if the row selected is the last movement the *_Question_* window is presented:
+
+image:image248.png[Question]
+
+Click *[.underline]##Y##es* if you want to confirm the deletion of the last movement or click *[.underline]##N##o* if you do not want to delete the movement. Before clicking the *[.underline]##Y##es* button check that the product present in the last movement is not used in the *_Pharmaceuticals Stock [.underline]##W##ard_*
+if it is then the *_Error_* window is shown.
+
+image:image249.png[Error]
+
+If it is not present then the last stock movement will be deleted from the table and the *_Success_* window is shown:
+
+image:image251.png[success]
+
+NOTE: This button can be deactivated to not allow movements to be deleted. Ask the Administrator or check the Administrator's Guide for more information.
```

Comment:
```suggestion
Click *[.underline]##Y##es* to confirm the deletion of the last movement, otherwise click *[.underline]##N##o* to cancel the operation. After clicking the *[.underline]##Y##es* button, the system checks that the product involved the selected movement has not been used itself in the *_Pharmaceuticals Stock [.underline]##W##ard_*.

If the product has been already used in a Ward, then the *_Error_* message is shown.

image:image249.png[Error]

Otherwise the selected movement is deleted and the *_Success_* window is shown:

image:image251.png[success]

NOTE: This button can be enabled/disabled to allow/not allow movements deletion. Ask the Administrator or check the Administrator's Guide for more information.
```